Task 2/Task 2.1.2/Task 2.1.2/User.cs

```diff
@@ -0,0 +1,42 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Task_2._1._2
+{
+    public class User
+    {
+        public User(string name)
+        {
+            Name = name;
+        }
+
+        private string _name;
+
+        public string Name
+        {
+            get => _name;
+            set => _name = value;
+        }
+
+        private List<object> _figures = new List<object>();
+
+        public List<object> ShowFigures()
+        {
+            return _figures;
+        }
+
```

Comment:
It doesn't show figures. This could be a property
```suggestion
        public List<object> Figures => _figures;
```